libbeat/docs/security/users.asciidoc

```diff
@@ -0,0 +1,295 @@
+indices.
+
+To assign the most restrictive privileges to users, create a role for each
+purpose. The easiest way to create roles is by using the
+*Management > Security > Roles* UI in {kib}.
+
+Also consider using the <<built-in-beats-roles,built-in roles>> provided by
+{security}.
+
+Want to learn more about creating users and roles? See
+{stack-ov}/elasticsearch-security.html[Securing the {stack}]. Also see
+{stack-ov}/security-privileges.html[Security privileges] for a description of
+available privileges.
+
+// REVIEWERS: I had to grant `monitor` to users to avoid errors, even though I
+// didn't enable monitoring. Is this expected? I've added `monitor` to the
+// privileges described below. Please let me know how I should handle this in
+// the docs.
+
+[[privileges-to-setup-beats]]
+==== Privileges needed for initial setup
+
+Users who set up {beatname_uc} typically need to load mappings, dashboards, and
+other objects used to index data into {es} and visualize it in {kib}. The
+privileges required depend on the setup tasks users need to perform:
+
+[options="header"]
+|====
+|Task | Required privileges
+
+|Set up index templates
+|`manage_index_templates` and `monitor` on cluster
+
+ifndef::no_dashboards[]
+.2+|Set up example dashboards
+|`monitor` on cluster
+
+|`index` on `.kibana*` indices
+endif::no_dashboards[]
+
+ifdef::has_ml_jobs[]
+.3+|Set up machine learning job configurations 
+|`manage_ml` and `monitor` on cluster
+
+|`read` on +{beat_default_index_prefix}-*+ indices
+
+|`read` and `index` on `.kibana*` indices
+endif::has_ml_jobs[]
+
+ifeval::["{beatname_lc}"=="filebeat"]
+|Set up ingest pipelines
+|`manage_ingest_pipelines` and `monitor` on cluster
+endif::[]
+
+ifndef::no_ilm[]
+|Set up index lifecycle policies
+|`manage_ilm` and `monitor` on cluster
+endif::no_ilm[]
+
+ifdef::has_central_config[]
+|Enroll and manage configurations in Beats central management
+|`all` on `.management-beats` index
+endif::has_central_config[]
+|====
+
+// REVIEWERS: We currently have manage_pipeline and
+// manage_ingest_pipelines. How are they different? Which is the correct one to
+// use here?
+
+[[privileges-to-publish-monitoring]]
+==== Privileges needed to publish monitoring information
+
+{security} provides the +{beat_default_index_prefix}_system+
+{stack-ov}/built-in-users.html[built-in user] and
++{beat_default_index_prefix}_system+ {stack-ov}/built-in-roles.html[built-in
+role] for sending monitoring information. You can use the built-in user, or
+create a user who has the privileges needed to send monitoring information.
+If you use the +{beat_default_index_prefix}_system+ user, make sure you
+<<beats-system-user,set the password>>.
+
+[options="header"]
+|====
+|Task | Required privileges
+
+|Send monitoring info | `monitor` on cluster
+|====
+
+// REVIEWERS: I don't have time to test the full monitoring set up. Is this
+// correct, or are additional privileges required?
```

Comment:
I think the `monitor` cluster privilege is for reading cluster state info, not for publishing stats.
I'm not even sure the privilege used by beats_system is exposed.